python/ray/train/torch/torch_trainer.py

```diff
@@ -22,13 +22,14 @@ class TorchTrainer(DataParallelTrainer):
     The ``train_loop_per_worker`` function is expected to take in either 0 or 1
     arguments:
 
-    .. code-block:: python
+    .. testcode::
 
         def train_loop_per_worker():
             ...
 
-    .. code-block:: python
+    .. testcode::
 
+        from typing import Dict
         def train_loop_per_worker(config: Dict):
```

Comment:
Ideally this would have a bit more typing like `Dict[str, Any]` (not sure what exactly the format here is) and also link to the format of the dict if possible :)

Reply:
Yeah, we can add some typing.